### PR TITLE
Listview behaviour

### PR DESCRIPTION
**Pull request by jeffersoncasimir**
Dragon slayed. Please test before merging.

This commit finally introduces the proper behaviour for our widget. We can add stocks to the ListView and see the change immediately.

Heavily inspired from Gurk's latest branch.

### CODE REVIEW

File: src/main/java/nitezh/ministock/activities/widget/WidgetView.java [hunk 1]
```diff
@@ -64,6 +68,7 @@ of this software and associated documentation files (the "Software"), to deal
     private final String quotesTimeStamp;
     private final Context context;
     private final HashMap<ViewType, Boolean> enabledViews;
+    public MyData theFkingList = new MyData();
```
Comment (Nick9500) [inline src/main/java/nitezh/ministock/activities/widget/WidgetView.java]: Maybe change the name to something more kosher?

File: src/main/java/nitezh/ministock/activities/widget/WidgetView.java [hunk 2]
```diff
@@ -80,7 +85,8 @@ public WidgetView(Context context, int appWidgetId, UpdateType updateMode,
                 PreferenceStorage.getInstance(context), widgetRepository).getStocksForSymbols(symbols);
         this.hasPortfolioData = !portfolioStocks.isEmpty();
 
-        this.remoteViews = this.getBlankRemoteViews(this.widget, context.getPackageName());
+        //this.remoteViews = this.getBlankRemoteViews(this.widget, context.getPackageName());
```
Comment (Nick9500) [inline src/main/java/nitezh/ministock/activities/widget/WidgetView.java]: Remove commented out line

**Review by Nick9500 — CHANGES_REQUESTED** [review]
Good work Jeff! Couple of minor changes to do then I'll okay it!

File: src/main/java/nitezh/ministock/activities/widget/WidgetView.java (hunk 1 above)
Comment (jeffersoncasimir) [reply src/main/java/nitezh/ministock/activities/widget/WidgetView.java]: I agree. I didn't call it that, lol

**Review by Nick9500 — APPROVED** [review]

**Comment by Nick9500** [comment]
This pull request references Issue #12

**Pull request merged**